platform/web-security/pom.xml

```diff
@@ -160,41 +160,39 @@
                             org.motechproject.security.osgi.Activator
                         </Bundle-Activator>
                         <Export-Package>
-                            org.motechproject.security.annotations;version=${project.version},
                             org.motechproject.security.authentication;version=${project.version},
-                            org.motechproject.security.builder;version=${project.version},
                             org.motechproject.security.constants;version=${project.version},
                             org.motechproject.security.domain;version=${project.version},
-                            org.motechproject.security.ex;version=${project.version},
+                            org.motechproject.security.exception;version=${project.version},
                             org.motechproject.security.model;version=${project.version},
-                            org.motechproject.security.repository;version=${project.version},
                             org.motechproject.security.service;version=${project.version},
+                            org.motechproject.security.service.mds;version=${project.version},
```

Comment:
I believe we agreed a while ago on making the name of this package mds, not service.mds (service.mds is a bit redundant)